kiva/fonttools/font_manager.py

```diff
@@ -139,10 +139,15 @@ def set_default_weight(self, weight):
 
     def update_fonts(self, filenames):
```

Comment:
Technically, this accepts filenames and file directories because `scan_user_fonts` supports files and dirs. Do we want to make that explicit in the method docstring?